src/sagas/auth/signin/email.js

```diff
@@ -1,14 +1,14 @@
 import { call, put, takeEvery, takeLatest } from '@redux-saga/core/effects';
 
-import { startLoginPasswordAuth } from '../../../store/auth/actions';
-import { AUTH_METHODS } from '../../../constants';
+import { AUTH_METHODS } from 'consts';
```

Comment:
also mind that imports from the core constants.js looks like `consts` to avoid naming collisions with node's default `constants` module.